controls/datapager/visual-structure.md

```diff
@@ -19,16 +19,19 @@ The following image shows the anatomy of the DataPager.
 - `First Page Button`&mdash;Represents the first page button. This button brings the pager on the first page of data.
 - `Previous Page Button`&mdash;Represents the previous page button. This button decrements the DataPager's current page index and brings it on the previous page of data.
 - `Numeric Page Buttons`&mdash;Represents a set of buttons that lists the available pages and allows the user to go directly to the desired one.
-- `Ellipsis`&mdash;Represents ellipsis. The ellipsis appears when the pages are more than the rendered numeric buttons.
+- `Ellipsis`&mdash;The first and the last numeric buttons can display ellipsis instead of numbers when the pages are more than the rendered numeric buttons.
```

Comment:
```suggestion
- `Ellipsis`&mdash;The first and the last numeric buttons can display an ellipsis instead of numbers when the pages are more than the rendered numeric buttons.
```